### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ruthgrace**
added flight manual to Your First Code Contribution, as recommended by @lee-dohm (personal communication via Slack). I thought the flight manual description on the [atom README.md](https://github.com/atom/atom/blob/master/README.md) was perfect, so I've just copied it here.

### CODE REVIEW

**Comment by 50Wliu** [comment]
Hmm...looks like the README needs to be updated as well: the URL is now `http://flight-manual.atom.io` and the repo is `https://github.com/atom/flight-manual.atom.io`.  In addition only the web and PDF versions are available now.

**Comment by ruthgrace** [comment]
Fixed! I'll submit a separate PR to update the links on the main Atom README too.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -221,6 +221,8 @@ Unsure where to begin contributing to Atom? You can start by looking through the
 
 Both issue lists are sorted by total number of comments. While not perfect, number of comments is a reasonable proxy for impact a given change will have.
 
+If you want to read about using Atom or developing packages in Atom, the [Atom Flight Manual](http://flight-manual.atom.io) is free and available as a PDF or web version. You can find the source to the manual in [atom/docs](https://github.com/atom/flight-manual.atom.io).
```
Comment (50Wliu) [inline CONTRIBUTING.md]: I would reword this to `...is free and available online or as a PDF`.  The `[atom/docs]` text also needs to be updated.

**Comment by ruthgrace** [comment]
Nice catch on the atom/docs link text. Thanks! I made both changes.

File: CONTRIBUTING.md (hunk 1 above)
Comment (lee-dohm) [inline CONTRIBUTING.md]: We don't offer a PDF version anymore either: https://github.com/atom/flight-manual.atom.io/issues/186

**Comment by lee-dohm** [comment]
Awesome, thanks! And thank you for your perseverance :grinning:
